config.js

```diff
@@ -86,17 +86,7 @@ config.nextCore = {
     rulesURL : 'http://demo-dca-be-01:8080/perseo-core/rules',
     noticesURL : 'http://demo-dca-be-01:8080/perseo-core/events'
 };
-/**
- * NEXT EPL core options (with HA, only for slave)
- *
- * reportInterval is the time in milliseconds between reports (error logs)
- * about master being missing
- */
-config.master = {
```

Comment:
`config.master` entry should be removed from configuration.md to keep doc in sync.